File: How to write an SFTP file transfer client for WinForms/[C#]-How to write an SFTP file transfer client for WinForms/C#/Sftp/SftpLoginInfo.cs
namespace ClientSample.Sftp
{
    class SftpLoginInfo
    {
        public const string PrivateKey = "PrivateKey";
        public const string EnableCompression = "EnableCompression";

        public static void SaveConfig(SettingInfoBase settings)
        {
            settings.SaveProperty(PrivateKey);
            settings.SaveProperty(EnableCompression);
        }

        public static void LoadConfig(SettingInfoBase settings)
        {
            settings.LoadProperty(PrivateKey, string.Empty);
            settings.LoadProperty(EnableCompression, false);
        }
    }
}

File: How to write an SFTP file transfer client for WinForms/[VB]-How to write an SFTP file transfer client for WinForms/VB/Sftp/SftpLoginInfo.vb
Namespace ClientSample.Sftp
	Friend Class SftpLoginInfo
		Public Const PrivateKey As String = "PrivateKey"
		Public Const EnableCompression As String = "EnableCompression"

		Public Shared Sub SaveConfig(ByVal settings As SettingInfoBase)
			settings.SaveProperty(PrivateKey)
			settings.SaveProperty(EnableCompression)
		End Sub

		Public Shared Sub LoadConfig(ByVal settings As SettingInfoBase)
			settings.LoadProperty(PrivateKey, String.Empty)
			settings.LoadProperty(EnableCompression, False)
		End Sub
	End Class
End Namespace